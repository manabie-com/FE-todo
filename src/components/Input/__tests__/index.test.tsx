import * as React from 'react';
import { render } from '@testing-library/react';

import { Input } from '../index';

describe('<Input />', () => {
  it('should match snapshot', () => {
    const input = render(<Input />);
    expect(input.container.firstChild).toMatchSnapshot();
  });
});
