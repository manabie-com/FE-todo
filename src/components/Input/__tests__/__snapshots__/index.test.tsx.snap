// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should match snapshot 1`] = `
.c0 {
  padding: 8px 16px;
  border-radius: 6px;
  width: 100%;
  outline: none;
  border: 1px solid rgba(34,36,38,0.15);
}

.c0:focus {
  border-color: #85b7d9;
}

<input
  class="c0"
/>
`;
